docs/api.asciidoc

```diff
@@ -194,6 +193,49 @@ elasticapm.set_transaction_result('SUCCESS')
  * `override`: if `True` (the default), overrides any previously set result.
     If `False`, only sets the result if the result hasn't already been set.
 
+
+[float]
+[[api-get-transaction-id]]
+==== `elasticapm.get_transaction_id()`
+
+Get the id of the current transaction. Example:
+
+[source,python]
+----
+import elasticapm
+
+transaction_id = elasticapm.get_transaction_id()
+----
+
+
+[float]
+[[api-get-trace-id]]
+==== `elasticapm.get_trace_id()`
+
+Get the parent_id of the current transaction's trace. Example:
```

Comment:
```suggestion
Get the `trace_id` of the current transaction's trace. Example:
```

Pretty sure I commented on this in a previous review, maybe it was lost in a refactor or something

Reply:
Yeah, weird, not sure how I missed this one. I only resolved each conversation after applying the recommendation. Maybe there was just another instance besides the one you pointed out previously.